inductiva/coastal/coastal_area.py

```diff
@@ -126,6 +128,39 @@ def get_config_filename(self, simulator: simulators.Simulator):
     def create_input_files(self, simulator: simulators.Simulator):
         pass
 
+    def _check_valid_wave(self):
+        """Checks that the wave amplitude is valid.
+
+        Raises:
+            ValueError: If the wave amplitude is not valid.
+        """
+
+        depths_grid = self.bathymetry.depths_grid()
+
+        if self.wave_source_location == "W":
+            wave_boundary = depths_grid[0, :]
+
+        elif self.wave_source_location == "E":
+            wave_boundary = depths_grid[-1, :]
+
+        elif self.wave_source_location == "N":
+            wave_boundary = depths_grid[:, -1]
+
+        elif self.wave_source_location == "S":
+            wave_boundary = depths_grid[:, 0]
+
+        else:
+            raise ValueError("Invalid wave source location.")
```

Comment:
So this just validates if the options are not in "S, W, N, E", right? I thought the goal was to identify that the direction cannot be at the highest points of the bathymetry (the beach). Confirm me this.

Reply:
It does both

Reply:
Ok great catch! Just clarify the error message below to mention the possibilities.